code/modules/mob/living/say.dm

```diff
@@ -142,6 +142,15 @@ var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
 	log_say("[name]/[key] : [message]")
 	return 1
 
+/mob/living/carbon/human/say(message, bubble_type,var/list/spans = list(), sanitize = TRUE)
+	if(..())
+		spawn(0)
+			var/loops = min(length(message)/5,20)
+			for(var/i = 0 ; i<loops;i++)

```

Comment:
To clarify, this has been fixed in the branch.  It just doesn't show as fixed because the PR is closed.